packages/components/src/buttons/src/Button.tsx

```diff
@@ -252,28 +147,29 @@ function Button(props: ButtonProps, ref: ForwardedRef<HTMLElement>) {
                 }]
             ]}
         >
-            <As
-                {...filterDOMProps(otherProps, { propNames: additionalButtonHTMLAttributes })}
-                {...buttonProps}
-                {...renderProps}
-                onClick={handleClick}
-                // We know the ref type match, ignore the error
-                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                // @ts-expect-error
+            <RACButton
+                {...otherProps}
                 ref={ref}
                 slot={props.slot || undefined}
+                className={classNames}
+                style={style}
+                isDisabled={isDisabled}
                 data-loading={isLoading}
```

Comment:
There is the new isPending property that needs to be ignore from our button prop, since we renamed it to isLoading. It should set the data-pending, so this line is not needed anymore. 

refer to this example: https://react-spectrum.adobe.com/react-aria/Button.html#pending